examples/sampling/pyg/test.py

```diff
@@ -0,0 +1,110 @@
+
+import torch
+from torch_geometric.data import Data
+from torch_geometric.nn import SAGEConv
+from torch_geometric.loader import NeighborSampler
+import dgl.graphbolt as gb
+import torch.nn.functional as F
+
+class GraphSAGE(torch.nn.Module):
+    def __init__(self, in_channels, hidden_channels, out_channels):
+        super(GraphSAGE, self).__init__()
+        self.conv1 = SAGEConv(in_channels, hidden_channels)
+        self.conv2 = SAGEConv(hidden_channels, out_channels)
+
+    def forward(self, x, adjs):
+        for i, (edge_index, _, size) in enumerate(adjs):
+            x_target = x[:size[1]]  
+            if i == 0:
+                x = self.conv1((x, x_target), edge_index)
+                x = F.relu(x)
+            else:
+                x = self.conv2((x, x_target), edge_index)
+                x = F.relu(x)
+            if i != len(adjs) - 1:
+                x = F.dropout(x, p=0.5, training=self.training)
+        return x
+
+
+def load_data():
+    dataset = gb.BuiltinDataset("ogbn-arxiv").load()
+    graph = dataset.graph
+    features = dataset.feature
+    features_tensor = features.read('node', None, 'feat')
+    train_set = dataset.tasks[0].train_set
+    valid_set = dataset.tasks[0].validation_set
+    test_set = dataset.tasks[0].test_set
+    train_labels = train_set._items[1]  
+    valid_labels = valid_set._items[1]
+    test_labels = test_set._items[1]
+    graph = dataset.graph
+    num_classes = dataset.tasks[0].metadata["num_classes"]
+    csc_indptr = graph.csc_indptr
+    rows = torch.repeat_interleave(torch.arange(csc_indptr.size(0) - 1, device=csc_indptr.device), csc_indptr[1:] - csc_indptr[:-1])
+    indices = graph.indices
+    cols = indices
+    edge_index =  torch.stack([rows, cols], dim=0)
+    pyg_data = Data(x=features_tensor, edge_index=edge_index, y=train_labels)  
+    num_nodes = csc_indptr.size(0) - 1
```

Comment:
We already have api to get num_nodes https://github.com/dmlc/dgl/blob/master/python/dgl/graphbolt/impl/fused_csc_sampling_graph.py#L40

Reply:
got it, thanks!